handbook/dendron.sop.daily-journal.md

```diff
@@ -0,0 +1,61 @@
+---
+id: f143773d-3f97-4dbd-8518-603fbb3b0288
+title: Daily Journal
+desc: ''
+updated: 1623169920368
+created: 1623165719449
+---
+## Summary
+
+The daily journal is how we do standup. Instead of a synchronous daily meeting, we do asynchronous journal notes. 
+
+We use the following [[template|templates.standup]] for people's daily journals. 
+
+Daily journals for the current day should be filled out by 5PM local time or before 8AM Pacific time (whichever comes first). A daily journal for the next day should also be created at the same time.
```

Comment:
Standup notes are the most useful at the *end* of each day. I think you want that to be the case for Seattle as well (doesn't make sense to have your journal filled out by 8am of the day). On a particular day (say, the 9th morning) we should only consider the 8th standup notes *filled out*. 9th is expected to be started.

Reply:
got callout. will change